openslides_backend/action/actions/projector/create.py

```diff
@@ -28,5 +29,9 @@ class ProjectorCreateAction(CreateAction):
             "show_title",
             "show_logo",
             "show_clock",
+            "used_as_reference_projector_meeting_id",
         ],
+        additional_optional_fields={
+            "used_as_default_$_in_meeting_id": used_as_default_for_schema,
```

Comment:
I still think it should work correctly when used normally in the `optional_fields`. I will look into it when this PR is ready.